### PR TITLE
Allow cli to accept config file param

### PR DESCRIPTION
**Pull request by kayakyakr**
Your project's first PR!

### CODE REVIEW

**Comment by jaydenseric** [comment]
Thanks for the PR 🙂

Taking a look at it now.

File: bin/webpack-watch-server.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ const Liftoff = require('liftoff')
 const chalk = require('chalk')
 const { spawn } = require('child_process')
 
+var argv = require('minimist')(process.argv.slice(2))
```
Comment (jaydenseric) [inline bin/webpack-watch-server.js]: Can the `require` please be broken out to the top of the file with the others, and the `var argv` be replaced with `const args`, just before the `const WebpackWatchServer =`.

File: readme.md [hunk 2]
```diff
@@ -23,6 +23,8 @@ Add `"dev": "webpack-watch-server"` to your package.json scripts.
 
 Run `npm run dev` to start Webpack in watch mode. The server automatically starts and restarts after each build.
 
+Use `webpack-watch-server --config my-custom-config.js` to specify an alternative webpack config.
```
Comment (jaydenseric) [inline readme.md]: This addition should follow the first line under "Usage", and could incorporate the phrasing `to specify a custom Webpack config file path`. Is it possible to use `--config subdirectory/my-custom-config.js`?

**Review by jaydenseric — COMMENTED** [review]
If you specify a custom path for config that needs to be passed with Babel, how does that work?

File: readme.md (hunk 2 above)
Comment (kayakyakr) [reply readme.md]: Don't see why not.

**Comment by kayakyakr** [comment]
Re Babel: I'm not sure it would work any differently as before. Any custom config should be a valid webpack.config.js file just like the default.

**Comment by jaydenseric** [comment]
Webpack automatically loads a config file that ends in `.babel.js` using Babel with config scoped to that particular directory.

To account for that, I am using `liftoff` and `babel-register` [here](https://github.com/jaydenseric/webpack-watch-server/blob/master/bin/webpack-watch-server.js#L13). With this PR I am not sure that will happen for users that try `webpack-watch-server --config config/webpack.config.babel.js`. Maybe test it out with a config file that needs to be transpiled?

**Comment by kayakyakr** [comment]
OK, so it doesn't do it automatically, but providing the `--require` directive allows you to compile babel or coffee or whatever you need to do.

Would you like me to bump version?

**Comment by jaydenseric** [comment]
So I tested it out, and `webpack-watch-server --config foo/bar.config.babel.js` works just fine. You need to have the `.config.babel.js` end to the file name for it to automatically transpile.

With that discovery, I think it's best to remove the `--require` stuff and recommend users to use the file name convention.

**Comment by kayakyakr** [comment]
Huh, I had a different result, but if you're good with it, I can revert that commit.

**Comment by jaydenseric** [comment]
Available in [v1.2.0](https://github.com/jaydenseric/webpack-watch-server/releases/tag/v1.2.0).

**Pull request merged**